distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -571,4 +571,12 @@ grant {
     permission java.io.FilePermission "/etc/hosts", "read";
     permission java.io.FilePermission "/etc/hostname", "read";
     permission java.io.FilePermission "/etc/resolv.conf", "read";
+
+    //Internationalization resource bundles
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}i18n", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}i18n${/}-", "read";
 };
+
+grant codeBase "file:/resource-bundle-locator/landing-page/catalog-ui-search" {
+    permission java.lang.RuntimePermission "createClassLoader";
```

Comment:
❗️ Absolutely right, for create classloader, that service should be the only one by using `doPrivileged()`. For the read permissions, I don't think these should be given to all or extended by the resource bundle locator since the data contained there is, by designed, provided to the client of that service and therefore, the client should have the permissions to read these files. Either that or the resource bundle locator should create its own new ResourcePermission and check for that and hide the file access under a `doPrivileged()` block.

Reply:
Permissions are now only granted to `resource-bundle-locator`